src/glb-director-xdp/bpf/bpf_helper_defs.h

```diff
@@ -0,0 +1,2819 @@
+/* This is auto-generated file. See bpf_helpers_doc.py for details. */
```

Comment:
Annoyingly, not packaged as part of libbpf's packaging. https://github.com/libbpf/libbpf/blob/master/src/bpf_helper_defs.h